services/storage/.storage.env

```diff
@@ -64,36 +64,26 @@ NEOFS_TREE_ENABLED=True
 #NEOFS_REPLICATOR_PUT_TIMEOUT=1m
 
 # Storage engine configuration
-
-## Default config for all shards
-### Write cache config
-NEOFS_STORAGE_SHARD_DEFAULT_WRITECACHE_ENABLED=false
-### Metabase config
-NEOFS_STORAGE_SHARD_DEFAULT_METABASE_PERM=0700
-### Blobstor config
-NEOFS_STORAGE_SHARD_DEFAULT_BLOBSTOR_PERM=0700
-NEOFS_STORAGE_SHARD_DEFAULT_BLOBSTOR_DEPTH=4
-NEOFS_STORAGE_SHARD_DEFAULT_BLOBSTOR_COMPRESS=true
-NEOFS_STORAGE_SHARD_DEFAULT_BLOBSTOR_SMALL_OBJECT_SIZE=102400
-### Blobovnicza config
-NEOFS_STORAGE_SHARD_DEFAULT_BLOBSTOR_BLOBOVNICZA_SIZE=134217728   # 128 * 1024 *1024
-NEOFS_STORAGE_SHARD_DEFAULT_BLOBSTOR_BLOBOVNICZA_DEPTH=1
-NEOFS_STORAGE_SHARD_DEFAULT_BLOBSTOR_BLOBOVNICZA_WIDTH=2
-NEOFS_STORAGE_SHARD_DEFAULT_BLOBSTOR_BLOBOVNICZA_OPENED_CACHE_CAPACITY=50
-### GC config
-#### Limit of the single data remover's batching operation in number of objects
-NEOFS_STORAGE_SHARD_DEFAULT_GC_REMOVER_BATCH_SIZE=100
-#### Sleep interval between data remover tacts
-NEOFS_STORAGE_SHARD_DEFAULT_GC_REMOVER_SLEEP_INTERVAL=1m
-
-## First shard
-### Metabase config
+NEOFS_STORAGE_SHARD_0_WRITECACHE_ENABLED=false
+NEOFS_STORAGE_SHARD_0_WRITECACHE_PATH=/storage/wc0
 NEOFS_STORAGE_SHARD_0_METABASE_PATH=/storage/meta0
-### Blobstor config
-NEOFS_STORAGE_SHARD_0_BLOBSTOR_PATH=/storage/blobstor0
+NEOFS_STORAGE_SHARD_0_BLOBSTOR_0_TYPE=blobovnicza
+NEOFS_STORAGE_SHARD_0_BLOBSTOR_0_PATH=/storage/blobstor0
```

Comment:
how about changing the path to "blobovniczas0" or smth like that? all the storages are the same now so "blobstor0" says nothing

Reply:
Fixed